### ISSUE TITLE
Force locale for XGPS and XATT messages format

### ISSUE THREAD

**Issue by NasH5169** [nguyenquyhy/Flight-Broadcaster#4]
Hi!

The culture of your app is not forcing a locale or at least decimal separator.

Some locales have the comma as decimal separator.

`XGPSFS2020,-0,6929301344339017,46,907469869333205,157,68469518029042,215,0740754322622,0,002090867443865422`

I think it is not really complicated to fix.

https://docs.microsoft.com/en-US/aspnet/core/fundamentals/localization?view=aspnetcore-5.0

**Comment by nguyenquyhy** [comment]
Yes, this tool is a bit outdated. I would suggest you to use Flight Events client instead https://events.flighttracker.tech. It works better with ForeFlight and has other features.

I will port the bug fix back here later.
